charactersheet/charactersheet/utilities/fixtures.js

```diff
@@ -26,7 +26,7 @@ var Fixtures = {
     },
     armor : {
         armorTypeOptions : [
-            'Light', 'Medium', 'Heavy', 'Shields'],
+            'Light', 'Medium', 'Heavy', 'Shield'],
```

Comment:
No, existing data will just not work. @jkrooskos and I discussed this and said it wasn't worth doing.